core/src/main/java/tc/oc/pgm/consumable/ConsumableDefinition.java

```diff
@@ -0,0 +1,21 @@
+package tc.oc.pgm.consumable;
+
+import org.jetbrains.annotations.Nullable;
+import tc.oc.pgm.features.SelfIdentifyingFeatureDefinition;
+import tc.oc.pgm.kits.Kit;
+
+public class ConsumableDefinition extends SelfIdentifyingFeatureDefinition {
+
+  protected final Kit kit;
```

Comment:
Use an Action<MatchPlayer> instead of a kit, generally speaking we're going to "deprecate" kit within pgm features, in favor of player-scoped actions. All kits are actions so you'll always be able to use a kit anywhere that expects a player-scoped action, but not all actions are kits.